plugin/src/node-builder.ts

```diff
@@ -4,12 +4,16 @@ import { createRemoteFileNode } from "gatsby-source-filesystem";
 // 'gid://shopify/Metafield/6936247730264'
 export const pattern = /^gid:\/\/shopify\/(\w+)\/(.+)$/;
 
-function attachParentId(obj: Record<string, any>, gatsbyApi: SourceNodesArgs) {
+function attachParentId(
+  pluginOptions: ShopifyPluginOptions, 
+  obj: Record<string, any>, 
+  gatsbyApi: SourceNodesArgs
+) {
```

Comment:
Somewhat nitpicky but I'd like `pluginOptions` to be the third argument here rather than the first.

Reply:
Fixed!